### DISCUSSION TITLE
License to use the data

### DISCUSSION THREAD

**Discussion by ghost** [TarkovTracker/tarkovdata#207] [Q&A] [answered]
Hey, I just discovered your repository but I don't see a license that allows me to use the data in the repository. Is that intentional that there is no license?

**Comment by kokarn** [comment, marked as answer]
Yes, sort of. It's free for anyone to use as they see fit :)

Maybe adding MIT would make that even more clear

**Comment by thaddeus** [comment]
Putting a license on it is weird, because its sort of Battlestate Games data, sort of public information, and also sort of a wiki meant for programs. But yes, as Kokarn mentioned, all the data is free to use
